src/Raven.Server/Smuggler/Documents/CsvStreamSource.cs

```diff
@@ -153,13 +155,26 @@ public DatabaseRecord GetDatabaseRecord()
 
         public IEnumerable<DocumentItem> GetDocuments(List<string> collectionsToExport, INewDocumentActions actions)
         {
+            var line = 0;
             while (_csvReader.Read())
             {
+                line++;
+
                 if (ProcessFieldsIfNeeded())
                     continue;
 
                 var context = actions.GetContextForNewDocument();
-                yield return ConvertRecordToDocumentItem(context, _csvReader.Context.Record, _csvReaderFieldHeaders, _collection);
+                DocumentItem item = null;
+                try
+                {
+                    item = ConvertRecordToDocumentItem(context, _csvReader.Context.Record, _csvReaderFieldHeaders, _collection);
+                }
+                catch (Exception e)
+                {
+                    _result.AddError($"Fail to parse CSV line {line}, Error:{e.Message}");
+                    item = new DocumentItem();
```

Comment:
Why are you returning an empty document instead of calling `continue` ?

Reply:
https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs#L495